npm/private/npm_package.bzl

```diff
@@ -172,6 +175,13 @@ def npm_package(
 
     This target can be used as the `src` attribute to `npm_link_package`.
 
+    Every npm_package target has a sub target named after its name, which is `<name>.publish`, that can be run
+    to publish to an npm registry.
+
+    You can pass arguments to npm by escaping them from Bazel using a double-hyphen, for example:
```

Comment:
"Under the hood, this target run `npm publish. You can pass arguments to npm by escaping..."